build.gradle

```diff
@@ -4,9 +4,10 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.0.0-beta5'
+        classpath 'com.android.tools.build:gradle:2.1.0-alpha4'
```

Comment:
@markmcd I'm not useing 2.1.0 features directly, but latest Android Studio 2.1 preview requires `2.1.0-alpha4` gradle plugin.

I can revert to `2.0.0-beta7` because it's not critical. Pls let me know if I should.